src/main/java/me/zeroeightsix/kami/command/commands/EjectCommand.kt

```diff
@@ -0,0 +1,105 @@
+package me.zeroeightsix.kami.command.commands
+
+import me.zeroeightsix.kami.KamiMod
+import me.zeroeightsix.kami.command.Command
+import me.zeroeightsix.kami.command.syntax.ChunkBuilder
+import me.zeroeightsix.kami.command.syntax.parsers.EnumParser
+import me.zeroeightsix.kami.module.modules.player.InventoryManager
+import me.zeroeightsix.kami.util.MessageSendHelper.*
+import net.minecraft.item.Item
+
+/**
+ * Created by 20kdc on 17/02/2020.
+ * Updated by dominikaaaa on 17/02/20
+ * Modified for use with AutoEject by Xiaro on 19/07/2020
+ */
+class EjectCommand : Command("eject", ChunkBuilder().append("command", true, EnumParser(arrayOf("help", "+item", "-item", "=item", "list", "default", "clear"))).build()) {
+    override fun call(args: Array<String?>) {
+        val im = KamiMod.MODULE_MANAGER.getModuleT(InventoryManager::class.java)
+        if (im == null) {
+            sendErrorMessage("&cThe module is not available for some reason. Make sure the name you're calling is correct and that you have the module installed!!")
+            return
+        }
+        if (!im.autoEject.value) {
+            sendWarningMessage("&6Warning: AutoEject in ${im.name} module is not enabled!")
+            sendWarningMessage("These commands will still have effect, but will not visibly do anything.")
+        } else if (im.isDisabled) {
+            sendWarningMessage("&6Warning: The ${im.name} module is not enabled!")
+            sendWarningMessage("These commands will still have effect, but will not visibly do anything.")
+        }
+        when {
+            args[0] == null || args[0].equals("help", ignoreCase = true) -> {
+                sendChatMessage("""Available options: 
+  +item: Adds item to the list
+  -item: Removes item from the list
+  =item: Changes the list to only contain item
+  list: Prints the list of selected items
+  defaults: Resets the list to the default list
+  clear: Removes all items from the AutoEject item list""")
+            }
+
+            args[0]!!.startsWith("+", true) -> {
```

Comment:
are you sure these should be `!!`? won't they be null if they don't exist?

Reply:
Maybe I should put the non null check in an if statement not when statement

Reply:
There is a non null check above, see line 31